config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
ah, @mcastelino @jodh-intel if we are just going with mainline (note my feedback about distro versions though etc.), then obviously the default system name is fine ;-)